### PR TITLE
feat: create skeleton component

### PR DESCRIPTION
**Pull request by Yur16**
## Description
Implements the sketeleon component with variants

## Screenshot
ezgif-1-43653373c2

## Compliance
- [x] I have verified that this change complies with our code and contribution policies.
- [x] I have verified that this change does not cause regressions and does not affect other parts of the code.

### CODE REVIEW

**Comment by iurynogueira** [comment]
ℹ View your Storybook at https://62eab350a45bdb0a5818520e-eabbtxpfgj.chromatic.com/

File: projects/ion/src/lib/skeleton/skeleton.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    await sut({ variant: 'circular' } as IonSkeletonComponent);
+
+    const skeleton = screen.getByTestId('ion-skeleton');
+    expect(skeleton).toHaveStyle('border-radius: 50%');
+  });
+
+  it('should accepts a custom border radius', async () => {
+    await sut({ variant: 'rect', radius: 12 } as IonSkeletonComponent);
+
+    const skeleton = screen.getByTestId('ion-skeleton');
```
Comment (iurynogueira) [inline projects/ion/src/lib/skeleton/skeleton.component.spec.ts:50]:
```suggestion
    const skeleton = screen.getByTestId('ion-skeleton');
```

You can create a function to return this element

File: projects/ion/src/lib/skeleton/skeleton.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    expect(skeleton).toHaveStyle('border-radius: 12px');
```
Comment (iurynogueira) [inline projects/ion/src/lib/skeleton/skeleton.component.spec.ts:51]:
```suggestion
    expect(getSkeleton()).toHaveStyle('border-radius: 12px');
```

and use here

File: projects/ion/src/lib/skeleton/skeleton.component.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<span
+  [style.width.px]="width"
+  [style.height.px]="height"
+  [style.borderRadius]="getRadius()"
```
Comment (iurynogueira) [inline projects/ion/src/lib/skeleton/skeleton.component.html:4]: call a function here can come lose performance, my suggestion is add your logic here

**Review by iurynogueira — CHANGES_REQUESTED** [review]
Thanks for your contribution @Yur16

**Review by iurynogueira — APPROVED** [review]
uhul

**Review by jairochaves2 — APPROVED** [review]

**Pull request merged**